### PR TITLE
JAMES-2587 Package RabbitMQ product

### PR DESCRIPTION
**Pull request by trantienduchn**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -194,11 +195,80 @@ If you are using a a fresh installation of Docker, your DOCKER_HOST should be un
...
+To run this container :
+
+    $ docker run --hostname HOSTNAME -p "25:25" -p 80:80 -p "110:110" -p "143:143" -p "465:465" -p "587:587" -p "993:993" --link cassandra:cassandra --link rabbitmq:rabbitmq
+   --link elasticsearch:elasticsearch --link tika:tika --name james_run -t james_run
+
+Where :
+
+- HOSTNAME: is the hostname you want to give to your James container. This DNS entry will be used to send mail to your James server.
+
+You can add an optional port binding to port 8000, to expose the webadmin server. Please note that users are not authenticated on webadmin server, thus you should avoid exposing it in production.
```
Comment (chibenwa) [inline README.adoc]: This is no longer true. You should say "webadmin is not secure by default, unless you configure JWT athentication"

File: dockerfiles/compilation/java-8/compile.sh [hunk 2]
```diff
@@ -59,6 +60,17 @@ fi
...
 if [ $? -eq 0 ]; then
+   if [ -d "$CASSANDRA_RABBITMQ_DESTINATION" ]; then
+      echo "Copying cassandra JARs"
+      cp server/container/guice/cassandra-rabbitmq-guice/target/james-server-cassandra-rabbitmq-guice.jar $CASSANDRA_RABBITMQ_DESTINATION || true
+      cp -r server/container/guice/cassandra-rabbitmq-guice/target/james-server-cassandra-rabbitmq-guice.lib $CASSANDRA_RABBITMQ_DESTINATION || true
+      cp server/container/cli/target/james-server-cli.jar $CASSANDRA_RABBITMQ_DESTINATION || true
+      cp -r server/container/cli/target/james-server-cli.lib $CASSANDRA_RABBITMQ_DESTINATION || true
+
+      # cp server/container/guice/cassandra-ldap-guice/target/james-server-cassandra-rabbitmq-ldap-guice.jar $CASSANDRA_RABBITMQ_DESTINATION || true
+      # cp -r server/container/guice/cassandra-ldap-guice/target/james-server-cassandra-ldap-rabbitmq-guice.lib $CASSANDRA_RABBITMQ_DESTINATION || true
```
Comment (chibenwa) [inline dockerfiles/compilation/java-8/compile.sh]: Useless comments that could be removed?

File: src/site/markdown/server/install/guice-cassandra-rabbitmq.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+```
+
+Once everything is set up, you just have to run the jar with:
+
+```bash
+$ java -Dworking.directory=. -jar target/james-server-cassandra-rabbitmq-guice.jar
+```
+
+## Guice-cassandra-ldap
+### Note: this product is not supported yet
```
Comment (chibenwa) [inline src/site/markdown/server/install/guice-cassandra-rabbitmq.md]: You need to add links on the website sidebar + in wiring.xml page

Comment (trantienduchn) [reply src/site/markdown/server/install/guice-cassandra-rabbitmq.md]: I guess webside slidebar is In `site.xml`. But Idk where `wiring.xml` is located. I searched whole project!

**Comment by trantienduchn** [comment]
```
[08f2452e34a467dcaff4dd4e9e61b564eddfb85d] [ERROR] Errors: 
[08f2452e34a467dcaff4dd4e9e61b564eddfb85d] [ERROR]   JpaListingTest.testListPercentWildcardKOREA » Runtime java.io.IOException: Una...
[08f2452e34a467dcaff4dd4e9e61b564eddfb85d] [INFO] 
[08f2452e34a467dcaff4dd4e9e61b564eddfb85d] [ERROR] Tests run: 315, Failures: 0, Errors: 1, Skipped: 0
```
Test this please

**Comment by trantienduchn** [comment]
test this please

File: src/site/markdown/server/install/guice-cassandra-rabbitmq.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+directory for configuration from
+[dockerfiles/run/guice/cassandra-rabbitmq/destination/conf](https://github.com/apache/james-project/tree/master/dockerfiles/run/guice/cassandra-rabbitmq/destination/conf). You might need to adapt it to your needs.
+
+You also need to generate a keystore in your conf folder with the following command:
+
+```bash
+$ keytool -genkey -alias james -keyalg RSA -keystore conf/keystore
+```
+
+You need to have a Cassandra and an ElasticSearch instance running. You can either install the servers or launch them via docker:
```
Comment (aduprat) [inline src/site/markdown/server/install/guice-cassandra-rabbitmq.md]: also add RabbitMQ in running instances.

**Pull request merged**